pentaho-karaf-features/src/main/resources/pentaho-karaf-features-standard.xml

```diff
@@ -65,6 +65,7 @@
     <feature>pentaho-client-minimal</feature>
     <feature>pentaho-big-data-plugin-osgi</feature>
    <bundle>mvn:commons-collections/commons-collections/${commons.collections.version}</bundle>
+    <feature>pdi-platform</feature>
```

Comment:
@tkafalas, It looks strange for me that we include the platform inside the client tool. Can you help me to understand why we should include the platform inside client tool? Thank you!